src/core/components/ErrorPage/index.js

```diff
@@ -1,28 +1,35 @@
+/* @flow */
 import * as React from 'react';
-import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
 
 import log from 'core/logger';
 import { getErrorComponent as getErrorComponentDefault } from 'core/utils';
 import { loadErrorPage } from 'core/reducers/errorPage';
-
-
-export class ErrorPageBase extends React.Component {
-  static propTypes = {
-    children: PropTypes.node.isRequired,
-    dispatch: PropTypes.func.isRequired,
-    errorPage: PropTypes.object,
-    getErrorComponent: PropTypes.func,
-  }
-
+import type { ErrorPageState } from 'core/reducers/errorPage';
+import type { DispatchFunc } from 'core/types/redux';
+
+
+type Props = {|
+  children: React.Node,
+  dispatch: DispatchFunc,
+  errorPage: {
```

Comment:
What I meant in the last review is that this can be:

```js
type Props = {|
  errorPage: ErrorPageState,
  ...
|};
```

Reply:
ah... right 🤦‍♂️